src/app/apps/rup/mapa-camas/views/plan-indicaciones/alergias-paciente.html

```diff
@@ -0,0 +1,24 @@
+<ng-container *ngIf="(registrosAlergia$ | async) as alergias">
+    <div *ngIf="alergias.length" class="mr-1 bordered" justify="between">
+        <plex-label class="ml-2" titulo="Paciente alérgico" [tituloBold]="false" subtitulo="" type="warning" size="md">
+        </plex-label>
+        <plex-help size="md" btnType="warning" [cardSize]="cardSize" icon="alert-circle-outline">
+            <plex-table size="sm">
+                <tr *ngFor="let registro of alergias">
+                    <td>
+                        <plex-label grow="4" titulo="{{ registro.nombre }}" subtitulo="{{ registro.fechaInicio | fecha }}"></plex-label>
+                    </td>
+                    <td *ngIf="registro.evolucion?.length">
+                        <plex-label>
+                            <b class="mr-1">Evolución:</b>
+                            <div [innerHtml]="registro.evolucion"></div>
+                        </plex-label>
+                    </td>
+                    <td *ngIf="!registro.evolucion?.length">
+                        <plex-label>Sin observaciones</plex-label>
+                    </td>
```

Comment:
Que opinas de pasar esto a un pipe para ahorrarnos los ngIf ??

Reply:
Estaria bueno, pero la salida de los pipes tiene que ser un string. Y en el caso de que haya alguna evolucion esta tiene que salir dentro de una etuiqueta htmlInner, por lo que en la pantalla se van a ver todas las equiquetas html en lugar de solo el contenido.

Reply:
@martinebucarey @negro89 se puede obtener ese comportamiento bindeando el resultado del pipe a la directiva  outerHTML del contenedor: https://stackoverflow.com/questions/34504203/how-to-allow-html-in-return-of-angular2-pipe
Vi un PR de @MCele donde se puede aplicar la misma solución